### DISCUSSION TITLE
Agent Committee Meeting, tomorrow, Wednesday 8/10/2022

### DISCUSSION THREAD

**Discussion by lin-fred** [ArctosDB/arctos#6134] [Agent Committee]
Hey everyone,

Reminder the next Agent meeting is Wednesday the 10th at 1:30pm MTN time. You can see the Arctos calendar for details and zoom link. Please contact me if you have issues using the zoom information as it would have changed. Also please note, that if you can, we are meeting 30 minutes before usual so that we can start editing agents as a committee.

Take a look at the agents project page https://github.com/ArctosDB/arctos/projects/11 under the Agent Committee Discussion/Verbatim Agents. Look over the issues, we will be going over them in order from top to bottom. Please add any that you want to discuss!

Agenda will be here: https://docs.google.com/document/d/1IGdLLjAfoWnMB1RUXdDq6WrRoqznVcyCiSLasYylDO8/edit#
Looking forward to seeing everyone!

Lindsey

**Comment by droberts49** [comment]
I’m out of the office this week with my kiddos, so I’ll miss the meeting tomorrow!
Dawn
